### PR TITLE
1052 replace today with 24 hours

### PR DESCRIPTION
**Pull request by timgl**
## Changes

- Based on ticket. The today/yesterday options were only used about half and 1/3 respectively compared to "last week". I feel confident this is a better solution than what we have now
image

## Checklist
- [ ] All querysets/queries filter by Team (if applicable)
- [ ] Backend tests (if applicable)
- [ ] Cypress E2E tests (if applicable)

### CODE REVIEW

**Comment by mariusandra** [comment]
Hi, I'm slightly conflicted by this.

"Today" and "yesterday" are not the same things as "last 24h". In one case you might want to use one, in a different case you might care about the other.

For example you have a dashboard for the sales team showing the sales for today big and center. That's no longer possible unless you add `dStart` to the URL manually.

This list is also quite long:

image

I was checking something yesterday and went from "last week" until "last 90 days". And then I wanted to see further back. Then the only thing I could do was to click "all time" or "year to date". I would have liked a "last 120 days" and "last 180 days" option. In my previous cpy we always looked at the "last 450 days" view by default, as that would show a year plus about 2-3 months leading up to "this point last year". 

One idea is to keep all of the options (today, yesterday and last x h) and move the "last X" options into a separate submenu "Last ..."

image

... with two extra submenus: "current [day/week/month/quarter/year]" and "previous [day/week/month/quarter]"?


PS. I think we should just run "black" on everything and rebase PRs as needed. It's really hard to follow what's happening when a small change adds ~200 lines of noise to the PR.

**Comment by mariusandra** [comment]
I'll add one thing: the "last 24h/48h/week/etc" version of the date filter does feel cleaner. However I'm just worried some existing users will have already built some dashboards around the "today" and "yesterday" filters, which they can no longer edit.

**Comment by timgl** [comment]
I agree with above, however after spending multiple hours(!) on trying to get the Cascade component to work I've given up.

The Cascade component is very different from the normal Select one, so I had to completely redo the date filter, it doesn't have the ability to style itself like the other Select boxes etc. This was supposed to be a 10 minute fix and I've decide it's not worth the time investment from me right now.

I've added Today/Yesterday. It's made the list quite long but it's still useable. If someone else wants to have a go go ahead but I'm not convinced this is a super high prio item.

**Review by mariusandra — APPROVED** [review]
Good for me!

**Comment by mariusandra** [comment]
I might have been too fast to accept it. There's one extra thing that would be nice to have: it might be cool to also see a hourly view with "today" and "yesterday", though I think this was already before this PR.

Selecting "yesterday" gives me this:

image
